### DISCUSSION TITLE
change the PriceConverter.sol imported paths

### DISCUSSION THREAD

**Discussion by MostPow3rful** [Cyfrin/foundry-full-course-cu#1883] [General]
**GM @PatrickAlphaC** 
File: https://github.com/Cyfrin/remix-fund-me-cu/blob/main/FundMe.sol#L5
Video: Foundry 101 Course - Section 2 - EP 3

there is a wrong import path at here. must be changed to this :
```py
import {AggregatorV3Interface} from "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
```
actually must remove the `/shared/`
i made a PR on github repo: https://github.com/Cyfrin/remix-fund-me-cu/pull/6
**GN**

**Comment by EngrPips** [comment]
Patrick mentioned he would push a lot of updates soon, So this might actually be corrected in the updates that are coming soon enough.

**Reply by MostPow3rful** [reply]
GM. sounds good. i just wanted to update the course material for other people who are installing dependencies.

**Reply by EngrPips** [reply]
Good morning, my Friend. Thanks for the heads-up. It is a good one.
